DEVELOPER.md

```diff
@@ -419,3 +426,79 @@ docker tag wind-river/cloud-platform-deployment-manager:debug ${MY_REGISTRY}/win
 docker push ${MY_REGISTRY}/wind-river/cloud-platform-deployment-manager:latest
 docker push ${MY_REGISTRY}/wind-river/cloud-platform-deployment-manager:debug
 ```
+
+## Troubleshooting dep
+If you encounter issues with dep (eg, dep command hangs), check the following:
+
+#### ~/.cache directory
+
+Your home directory is a shared file system under /folk. Having the .cache
+directory in your folk directory can end up filling up the disk quota. Use a
```

Comment:
Remove the reference to "/folk", as this is specific to your environment.
"Your home dir may be a shared file system. If disk quotas are enabled due to limited space, the .cache directory could end up filling up your quota."

Reply:
Fixed